database-service/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM hasura/graphql-engine:v1.0.0-beta.8.cli-migrations AS app
+
+COPY migrations /app/migrations
+COPY config.yaml /app 
+
+WORKDIR /app
+
+FROM python:3.8-slim AS unittest
+
+RUN apt-get -y update \
+  && apt-get install -y curl \
+  && curl -L https://github.com/hasura/graphql-engine/raw/master/cli/get.sh | bash
+
+COPY tests/requirements.txt /tmp
+
+WORKDIR /tmp
+
+RUN pip install -r requirements.txt \
+  && rm requirements.txt 
+
+WORKDIR /app
```

Comment:
I've put the builds of all current configurations in one single dockerfile, as recommended by the docker community. Is that fine like that? can that dockerfile be optimized?

Reply:
See my answer to that below. For me, it's a matter of taste. The production docker image for the app needs its entrypoint in the `Dockerfile`, but anything else is a matter of taste.